data-raw/qc-markdowns/temperature_data_prep.Rmd

```diff
@@ -532,9 +534,10 @@ GRL_hourly_temps %>%
 
 ### USGS
```

Comment:
The remaining TODO in the Feather section is to apply a threshold criteria to the GRL data (I just looked at it and it looks really bad) - you can use similar approach as we did for Butte (USGS max/min). Then save the resulting daily GRL data as hfc_feather and the FEA as lfc_feather.